Tests/js/alloyeditor/buttons/assets/ez-alloyeditor-button-heading-tests.js

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
why did you add this file and ez-alloyeditor-toolbar-appendcontent-tests.js ? they were not added to git on purpose